administration/src/components/generation/GenerationForm.tsx

```diff
@@ -84,6 +84,7 @@ const GenerationForm = (props: Props) => {
                     {cardCreationModels.length === 0 && "Legen Sie zunächst eine Karte an."}
                 </Tooltip>
             </ButtonBar>
+            { /* @ts-ignore */}
```

Comment:
Ohoh.

Reply:
Yeah, the npm package react-flip-move is not maintained anymore and has wrong ts types... We should probably switch to a different library (eg. [react-flip-toolkit](https://github.com/aholachek/react-flip-toolkit).)